src/main/java/mate/jdbc/dao/interfaces/ManufacturerDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.dao.interfaces;
+
+import java.util.List;
+import java.util.Optional;
+import java.sql.SQLException;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer) throws SQLException;
+
+    Optional<Manufacturer> get(Long id) throws SQLException;
+
+    List<Manufacturer> getAll() throws SQLException;
+
+    Manufacturer update(Manufacturer manufacturer) throws SQLException;
+
+    boolean delete(Long id) throws SQLException;
+}
```

Comment:
```suggestion
Manufacturer create(Manufacturer manufacturer);

    Optional<Manufacturer> get(Long id);

    List<Manufacturer> getAll();

    Manufacturer update(Manufacturer manufacturer);

    boolean delete(Long id);
```
Впринципі, ми маємо опрацювати ці помилки при реалізації імпліментації цього інтерфейсу аби їх не писати в сигнатурах методів постійно